webcomponents/tool/src/main/frontend/sakai-i18n.js

```diff
@@ -71,10 +71,6 @@ function loadProperties(options) {
     params.append("resourceclass", options.resourceClass);
     params.append("resourcebundle", options.bundle);
 
-    var fetchPromise = fetch(`/sakai-ws/rest/i18n/getI18nProperties?${params.toString()}`, {
-      headers: { "Content-Type": "application/text" },
-    });
-
```

Comment:
@adrianfish i couldn't figure out what this line does .... i don't see "fetchPromise" used anywhere else

Reply:
Good spot. Redundant that be.